atuin-client/src/record/sqlite_store.rs

```diff
@@ -62,11 +63,11 @@ impl SqliteStore {
             "insert or ignore into records(id, host, tag, timestamp, parent, version, data, cek)
                 values(?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8)",
         )
-        .bind(r.id.as_str())
-        .bind(r.host.as_str())
+        .bind(r.id.as_simple().to_string())
+        .bind(r.host.as_simple().to_string())
```

Comment:
does sqlite not support 128 bit integers? :(

Reply:
it does not :(